File: Pratica 5 - Funcoes/cpp_logic/exemplo1.cpp
#include <iostream>
#include <string>

void exibirMenu() {
    std::cout << "---------------------------------" << std::endl;
    std::cout << "     MENU DE ESTOQUE     " << std::endl;
    std::cout << "---------------------------------" << std::endl;
    std::cout << "1. Adicionar novo produto" << std::endl;
    std::cout << "2. Listar produtos em estoque" << std::endl;
    std::cout << "3. Sair" << std::endl;
    std::cout << "---------------------------------" << std::endl;
}//Bloco da função encolhida

int main() {
    std::string nome_produto1 = "";
    int quant_produto1 = 0;
    double preco_produto1 = 0.0;
    bool rodando = true;

    while (rodando) {
        exibirMenu(); //Chamada da função
        //continuação do programa
    }
}

File: Pratica 5 - Funcoes/java_logic/Exemplo1.java
import java.util.Scanner;

public class Exemplo1 {
    public static void exibirMenu() {
        System.out.println("---------------------------------");
        System.out.println("     MENU DE ESTOQUE     ");
        System.out.println("---------------------------------");
        System.out.println("1. Adicionar novo produto");
        System.out.println("2. Listar produtos em estoque");
        System.out.println("3. Sair");
        System.out.println("---------------------------------");
    } //Bloco encolhido
    
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        String nome_produto1 = "";
        int quant_produto1 = 0;
        double valor_produto1 = 0.0;
        boolean rodando = true;

        while (rodando){
            exibirMenu(); //Chamada da função
            //Continuação do código
        }
    }
}
